### ISSUE TITLE
 You have requested a non-existent service "group_context_domain".

### ISSUE THREAD

**Issue by finnlewis** [localgovdrupal/localgov_microsites_group#466]
Trying to submit a webform on a microsite is giving me this error:

```
Drupal\Core\Entity\EntityStorageException: You have requested a non-existent service "group_context_domain". Did you mean this: "group_context_domain.group_from_domain_context"? in Drupal\Core\Entity\Sql\SqlContentEntityStorage->save() (line 817 of core/lib/Drupal/Core/Entity/Sql/SqlContentEntityStorage.php). 
```

**Comment by ekes** [comment]
https://github.com/localgovdrupal/localgov_microsites_group/pull/467

**Comment by ekes** [comment]
Fixed in #467
